server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPConfiguration.java

```diff
@@ -114,11 +115,16 @@ public Builder maximumSendSize(Optional<Long> maximumSendSize) {
             return this;
         }
 
+        public Builder supportsDelaySends(Optional<Boolean> supportsDelaySends) {
+            this.supportsDelaySends = supportsDelaySends;
+            return this;
+        }
+
         public JMAPConfiguration build() {
             Preconditions.checkState(enabled.isPresent(), "You should specify if JMAP server should be started");
             return new JMAPConfiguration(enabled.get(), port, emailQueryViewEnabled.orElse(false),
                 userProvisioningEnabled.orElse(true),
-                defaultVersion.orElse(Version.DRAFT), maximumSendSize);
+                defaultVersion.orElse(Version.DRAFT), maximumSendSize, supportsDelaySends.get());
```

Comment:
Never use optional.get directly. If it is an `Optional.empty`, you would get a `NullPointerException`.

Suggestion: use `.orElse(defaultValue)`.